### DISCUSSION TITLE
Add WaybackProxy

### DISCUSSION THREAD

**Discussion by kyousif87** [retronas/discussions#20] [Ideas]
Hello there,

Is it possible to add WaybackProx https://github.com/richardg867/WaybackProxy to the repertoire of stuff that RetroNAS has? :)

Thanks,
Khalid

**Comment by sairuk** [comment]
Seems like it might be a good fit, converted this issue to an Ideas discussion

**Reply by sairuk** [reply]
waybackproxy has been added to the testing branch, thanks for the suggestion

**Reply by kyousif87** [reply]
my pleasure!

Thank you for adding it to the testing branch

will attempt to test it tomorrow and report back :)
